app/models/public_body.rb

```diff
@@ -217,15 +217,19 @@ def set_api_key!
   end
 
   def self.find_by_name(name)
-    PublicBody.joins(:translations).
+    found = PublicBody.joins(:translations).
       where("public_body_translations.name = ?", name).
        first
+
+    PublicBody.find(found.id)
```

Comment:
It's possible the `default_scope` is doing harm here as adding `unscoped` to the query seems to help, and adding `with_translations` appears to ensure the translations are loaded (the localisation integration spec appears to be the best test for this)

Looking into removing default_scope on that basis, but may keep the `unscoped` syntax for safety.